aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -933,11 +953,16 @@ module AptosFramework::Stake {
         remaining_lockup_time: u64,
         validator_set_config: &ValidatorSetConfiguration,
     ): u64 acquires TestCoinCapabilities {
+        let stake_amount = Coin::value<TestCoin>(stake);
```

Comment:
it’s probably cleaner to use the voting power to calculate the rewards instead of having two places doing the floor operation

Reply:
That could work. It'd change the behavior here a bit though - we can calculate the rewards and also keep that in active instead of giving to both active and pending_inactive like what we have currently.

Reply:
i think it’s reasonable to accumulate rewards to active stake

Reply:
I updated the logic to issue rewards based on voting power to active if validator has remaining active stake > 0 or pending_inactive otherwise. This is a simpler behavior that before (calculating rewards separate for active and pending_inactive). The only edge cases here are:

1. Users have dust left in active. This should be easily solvable via UI or dependent contracts where there's a withdraw all option that withdraws all stake.
2. If later we disallow withdrawing beyond min required stake if the validator is still active, the UX might be strange as the active stake would keep growing even though the validator can't withdraw past the min required amount. This can be easily solved by having the validator leave the set first before unlocking/withdrawing. So this in general is a problem introduced by the withdrawal limit, not by changing where rewards are distributed.